src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,21 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
+
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @ManyToMany
```

Comment:
Remember to specify the details of the @ManyToMany relationship. Without specifying 'mappedBy' or defining a 'join table', Hibernate may not create the relationship correctly. Review the relationship setup to ensure it reflects the intended database design.